### ISSUE TITLE
A new window should open on the current workspace, if not already existing

### ISSUE THREAD

**Issue by alex285** [elementary/files#964]
This is mostly a Gala and Plank issue, it is a general design concept, but is also related to https://github.com/elementary/files/commit/f0b1f356cbfba0460a1d90df9f7fbcacd238ba5d so I open it here

On the simpler form, assume we have two workspaces, wA and wB. On wA there is a Files window running, and we are currently working on wB

When we open a new Files window, either with Ctrl+Click from Plank, or with "Show in File Browser" from Terminal app, elementary will open a new Files tab on wA and it will switch us workspace

I believe that the correct behavior is: 
A. When there is a Files window on the current workspace, then open a tab
B. When there is NOT a Files window on the current workspace, then open a new window

I'm not very sure if that explained it very well, but I demo that particular issue on this video:
https://youtu.be/aHcKYOYM5wQ

**Comment by jeremypw** [comment]
I am not sure if there is a way Files can know or influence which workspace its windows are displayed on

**Comment by alex285** [comment]
Without having opinion whats the "most optimal" if it is not possible to know on what workspace each window is, I think at least the Ctrl+Click on Plank should always open a new window, since this is most expected behavior, as the same mechanism is on Windows10 (w Shift), GNOME and KDE

Also it makes more sense, because if you already have a window on the current workspace, there is not much point to open another; You can directly open a new tab from the Files app

**Comment by jeremypw** [comment]
I agree that `<Ctrl><Click>` in Plank should give the same result as the "New Window" Plank context menu option.  I thought that is already the case and the PR you mentioned will not change that.

**Comment by alex285** [comment]
>I thought that is already the case and the PR you mentioned will not change that.
yeap, i know, that issue isn't actually fit on Files, however Plank also opens a new tab, instead of a new window; i didn't build the latest, i used elementary nightly PPA, and i don't know if something's has changed, but on 2:06 of that video I shared thats happening: https://youtu.be/aHcKYOYM5wQ?t=122

**Comment by jeremypw** [comment]
Closing as I do not think this is fundamentally a Files issue and Planck is no longer used.  Feel free to re-open if a similar issue can be identified on OS8 that can be fixed within Files.
